integration/js/AudioTest.js

```diff
@@ -21,36 +21,70 @@ class AudioTest extends SdkBaseTest {
   }
 
   async runIntegrationTest() {
+    this.numberOfParticipant = 2;
+
+    if (this.numberOfSessions() > 1) {
```

Comment:
These 2 sessions are specially for `safari` and `ANDROID`?

Reply:
No, KITE uses it to generate some logs or do some synchronization.

e.g. https://github.com/aws/amazon-chime-sdk-js/blob/d1a6ed82220a70e6d5aa576c09d5fe28cf0526da/integration/js/utils/SdkBaseTest.js#L216-L225